### ISSUE TITLE
Should we retire this repo?

### ISSUE THREAD

**Issue by syg** [tc39/ecmascript_sharedmem#170]
Now that SAB has been merged into ecma262 main spec text, should we retire this repo?

It may be valuable to track SAB issues separately here since it cross-cuts DOM and ECMAScript. So I think there are two options:

1. Move master branch to an archived branch. Remove all spec text with a README redirecting the reader to ecma262 and the DOM specs. Keep repo around for filing issues.
2. Retire the whole repo. Migrate outstanding issues to various DOM spec repos and ecma262.

Thoughts?

**Comment by lars-t-hansen** [comment]
@syg, I think option (1) is the better.  Most of the things labeled as DOM bugs here have counterparts in the various whatwg and sometimes ecma262 repos, but can usefully stay open until matters have been resolved.  What's labeled Future work should remain visible, IMO.

(I've closed everything not in those two categories, since the feature is baked, browsers are shipping it, and the issues were actually resolved.)
